valohai/outputs.py

```diff
@@ -20,20 +20,21 @@ def __init__(self, name: str = "") -> None:
     def path(self, filename: str, makedirs: bool = True) -> str:
         """Absolute path of an output file
 
-        Compress files into a temporary file, which is then copied to the target path.
-        Optionally removes original files and live uploads the package.
```

Comment:
I guess this was just entirely bogus here?

Reply:
Yeah, this was an accidental copy-paste leftover that made zero sense.